src/components/ProgressBars/LinearProgressBar/__stories__/linearProgressBar.stories.mdx

```diff
@@ -0,0 +1,214 @@
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Progress bars show continuous progress through a process, such as a percentage value. They show how much progress is complete and how much remains.
+<Canvas>
+  <Story name="Overview"
+         args={ {value: 20, size: LinearProgressBar.sizes.LARGE} }>
+    { lineProgressBarTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ LinearProgressBar } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Give users an indication of how much of the task has been completed and how much is left."
+]}/>
+
+<Tip>If you need to lead a user through a progress, use the <Link href="/?path=/docs/components-Wizard--overview" size={Link.sizes.SMALL} withoutSpacing>Wizard</Link> instead.</Tip>
+
+## Variants
+### Regular
+<Canvas>
+  <Story name="Regular">
+    <div className="linear-progress-bar_small-wrapper">
+      <div className="linear-progress-bar_block">
+        <LinearProgressBar indicateProgress value={30} size={LinearProgressBar.sizes.LARGE} />
+        <>With label</>
+      </div>
+      <div className="linear-progress-bar_block">
+        <LinearProgressBar value={30} size={LinearProgressBar.sizes.LARGE} />
+        <>Without label</>
+      </div>
+    </div>
+  </Story>
+</Canvas>
+
+### With secondary value
+<Canvas>
+  <Story name="With secondary value">
+    <div className="linear-progress-bar_small-wrapper">
+      <LinearProgressBar value={50} indicateProgress valueSecondary={65} size={LinearProgressBar.sizes.LARGE} />
+    </div>
+  </Story>
+</Canvas>
+
+### Multi progress bar
+<Canvas>
+  <Story name="Multi progress bar">
+    {() => {
+      const multiValues = [
+        { value: 25, color: colors.primary },
+        { value: 75, color: colors.yellow },
+        { value: 100, color: colors.grass_green }
+      ];
+      return (
+        <div className="linear-progress-bar_big-wrapper">
+          <LinearProgressBar 
+            value={25}
+            size={LinearProgressBar.sizes.LARGE}
+            indicateProgress
+            multi
+            multiValues={multiValues}
+          />
+        </div>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+          <DialogContentContainer className="linear-progress-bar_dialog-wrapper">
+            <h4>Items usage</h4>
+            <div className="linear-progress-bar_inline-wrapper">
+              <span>Items</span>
+              <span>142/200</span>
+            </div>
+            <LinearProgressBar value={71} />
+          </DialogContentContainer>,
+        description:"Use a progress bar only process has start and finish point."
+      },
+      negative: {
+        component:
+          <DialogContentContainer className="linear-progress-bar_dialog-wrapper">
+            <h4>Storage</h4>
+            <div>
```

Comment:
If you can remove this div without change the negative example appearance please remove it